packages/palette-docs/content/docs/tokens/Icons.mdx

```diff
@@ -12,7 +12,7 @@ for the masterlist.
 
 <Flex flexWrap="wrap">
   {Object.entries(AllIcons).map(([iconName, IconComponent], index) => {
-    const blacklist = ["ChevronIcon", "CreditCardIcon", "Icon"]
+    const blacklist = ["ChevronIcon", "CreditCardIcon", "Icon", "G", "Title", "Circle", "Path", "Rect"]
```

Comment:
While you're here, how would you like to update this name to something nicer like `blocklist`?